repaygithub/cactus
# CACTUS-616 :: Header Description

Ticket: https://repayonline.atlassian.net/browse/CACTUS-616

I'm still not sure if the `text` prop really provides any value. I just added it because I thought it might be nice to offer a couple of different ways of setting it up, but maybe it's not even worth including it. I dunno, I could go either way. Let me know what you guys think.

I didn't add any unit tests 'cause I figure the storyshots should cover it anyway.

The Header docs were pretty broken -- no way to try it out on the page, props tables passing the wrong `displayName`s, that sort of thing. I fixed those up, though.

Still looks really bad in IE on mobile, but I guess that's just something we're living with, for now anyway.

### Testing
1. Pull down this branch and run the storybooks
2. Check out the "With Description" and "With Everything" stories. Make sure it all looks good. Add some text to make sure it wraps properly.
3. Change the screen size to make sure it looks good at all screen sizes.
4. Check it out in other browsers. Don't expect it to look good in IE/mobile.
